### ISSUE TITLE
readthedocs theme: .rst-content code color defined twice, all code is black

### ISSUE THREAD

**Issue by wodny** [mkdocs/mkdocs#2027]
After upgrading from mkdocs 1.0.4 to 1.1 all code blocks (including those inline) became black instead of still having the well known orange color. Browser inspector shows there are duplicated rules for `.rst-content code` color - `#000` and `#E74C3C`. Is this is a bug? In my opinion previous colors looked much better and online readthedocs documents seem to still use orange.

**Comment by waylan** [comment]
In the most recent version of MkDocs we updated to a more recent version of the upstream's CSS. With a quick look in the inspector...

![Screen Shot 2020-03-20 at 3 49 44 PM](https://user-images.githubusercontent.com/78846/77201471-e962fe00-6ac2-11ea-8635-c7881fa9c66b.png)

... we can see that both color definitions are found in `theme.css` which is the upstream file. We have made no edits to that file. However, we can also see that the MkDocs specific overrides in `theme_extra.css` do not override the color. We need to provide our own overrides for some things as the HTML output by MkDocs/Markdown is slightly different than the HTML output by Sphinx.  If we need to add something to the overrides in `theme_extra.css` to better match upstream, a PR is always welcome (such a PR would ideally provide street shots of both before and after in MkDocs as well as of the upstream Sphinx theme as justification for the change).

If you would like to know why the color is defined twice, that is a question to ask upstream.
